orbstation/quirks/xcard/bioscrambler_immune.dm

```diff
@@ -0,0 +1,61 @@
+//alternative nasty effects for the Bioscrambler Anomaly, for use in the x-card quirk and in species unsuitable for limb-swapping
+/obj/effect/anomaly/bioscrambler/proc/alt_swap(mob/living/carbon/human/victim)
+	var/bad_effect = rand(1,3)
+	var/zones
+	var/picked_zone
+	var/obj/item/bodypart/picked_part
+	switch(bad_effect)
+		if(1) //dismember random limb
```

Comment:
Stop me if this sounds too silly but you might be able to do:
```
switch(bad_effect)
   if(1 to 2)
      var/picked_zone = pick(list(BODY_ZONE_L_ARM, BODY_ZONE_R_ARM, BODY_ZONE_L_LEG, BODY_ZONE_R_LEG))
      var/picked_part = victim.get_bodypart(picked_zone)
      if(!picked_part)
         return
      if(1)
         ...
      if(2)
         ...
   if(3)
      ...
```

Reply:
1 and 2 are separate because the bone-breaking can target _any_ body part (including head and torso), but I didn't want limb-disintegration to target anything vital. They're _just_ different enough that I'm not sure there's much that can be combined.

Reply:
Didn't realise they had different lists, should have read more closely.